### PR TITLE
The Next Web Clone

### PR DESCRIPTION
**Pull request by ishanchawla1**
- Sticky Navbar with useful links and Logo
- Multiple sections like News, Deals, Funding
- Use of Grid and Flex in multiple sections
- Footer with social links
- Responsive Design for Mobile and Tablet

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,931 @@
...
+            <li><a href="#">Advertisement</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+        </ul>
+        </div>
+    <nav>
+        <div class="lower-nav">
+            <img src="https://smart.beamian.com/wp-content/uploads/2018/05/the_next_web_logo.jpg" alt="TNW_Logo"/>
+            <div class="adjust-flex">
+            <ul class="navSwitch">
```
Comment (majovanilla) [inline index.html:32]: The logo shouldn't appear on large screens.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,931 @@
...
+                       <img class="headerIcons" src="Images/clock.png" alt="Clock Icon"/>                           
+                       <span class="greyScale smallMarginLeft noStyle">1 hour ago</span>
+                       <img class="headerIcons" src="Images/share.png" alt="Share Icon"/>
+                       <span class="greyScale smallMarginLeft noStyle">12</span>
+                    </div>
+                </div>
+            </article>
+        </section>
+        <section class="latestNews">
+            <div class="latestNewsHeading redHeading"><h2>Latest news > </h2> </div>
```
Comment (majovanilla) [inline index.html]: The section is too wide compared to the original. That makes all the elements have larger dimensions. Please make the necessary adjustments.
Check the title of the latest news, too. The font-size is larger in your page.
Yours:
Screenshot from 2020-04-13 13-26-29

Original:
Screenshot from 2020-04-13 13-26-37

File: index.html [hunk 3]
```diff
@@ -0,0 +1,931 @@
...
+                </a>
+                </h2>
+                <a href="#" class="noTextDecoration authorName"> TNW Deals </a> <span class ="timeAgo">7 hours ago</span>
+                </div>
+            </article>
+        </section>
+        <section class="latestFunding">
+            <div class="latestFundingHeader">
+                <a href="#" class="redHeading noTextDecoration">
+                    Latest funding rounds on Index.co >
```
Comment (majovanilla) [inline index.html:232]: The title and the small menu are more separated from the section than they should. Please adjust the spacing.
This section is wider than it should be, too.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,931 @@
...
+                    <a href="#" class="companyInfo">
+                        Korean investors contribute $43m to India's Swiggy <span class="greyScale">on AVCJ</span>
+                    </a>
+                </div>
+            </div>
+        </section>
+        <section class="popularArticles">
+            <h2 class="switch">display none</h2>
+            <article class="populatArticle1">
+                <h2>
```
Comment (majovanilla) [inline index.html:363]: The categories section has a different width than the first sections. This shouldn't happen (they have the same width).

Also, the space between rows is too large. Please adjust it.
Yours:
Screenshot from 2020-04-13 13-29-59

Original:
Screenshot from 2020-04-13 13-30-04

File: index.html [hunk 5]
```diff
@@ -0,0 +1,931 @@
...
+                </div>      
+            </article>       
+        </section>
+        <section class="latestDeals">
+            <h2 class="switch">display none</h2>
+            <div class="sectionTitle">
+                <a href="#" class="redHeading noTextDecoration latestDealHeader">
+                    Latest Deals >
+                </a>
+                <a href="#" class="redHeading noTextDecoration seeAll">
```
Comment (majovanilla) [inline index.html:761]: The images in this section have the wrong height. Please adjust it.
Also, check the width of the whole section related to the page.
Yours:
Screenshot from 2020-04-13 13-32-28

Original:
Screenshot from 2020-04-13 13-32-34

File: index.html [hunk 6]
```diff
@@ -0,0 +1,931 @@
...
+            <p>ABOUT</p> 
+            <p>ADVERTISE</p> 
+            <p>JOBS</p> 
+            <p>CONTACT</p> 
+        </div>
+        <div class="footerSecond">
+            <a href="#"><span class="colorWhite">The Next Web </span></a> 
+            <span class="greyScale">© 2006–2020 The Next Web B.V. Made with ♥ in Amsterdam. Powered by </span> 
+            <a href="#"><span class="colorWhite">STACKPATH</span></a>
+        </div>
```
Comment (majovanilla) [inline index.html]: All the elements in the footer should be links. In the first half you have images, enclose them in `<a>` tags.
In the second part you have `<p>` tags, change them to be `<a>` tags.
This is the best practice when building a footer and you want each element to redirect to another place.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,931 @@
...
+            <p>EVENTS</p> 
+            <p>ABOUT</p> 
+            <p>ADVERTISE</p> 
+            <p>JOBS</p> 
+            <p>CONTACT</p> 
+        </div>
+        <div class="footerSecond">
+            <a href="#"><span class="colorWhite">The Next Web </span></a> 
+            <span class="greyScale">© 2006–2020 The Next Web B.V. Made with ♥ in Amsterdam. Powered by </span> 
+            <a href="#"><span class="colorWhite">STACKPATH</span></a>
```
Comment (majovanilla) [inline index.html]: Include the logos that the original page has.

**Review by majovanilla — CHANGES_REQUESTED** [review]
Hi @ishanchawla1 

## Good job on this project! 👍

- Good Readme file
- Good Pull Request Message

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Your page doesn't have the same breakpoints as the original page.
- You should take a closer look at how the elements change on each breakpoint, because yours are not behaving as expected: 
  - On mobile: almost all the articles are shown in one column.
  - On tablet: almost all the articles are shown in two columns.
  - On desktop you have the right structure but not the right width for some parts (check details below)
Yours on medium:
Screenshot from 2020-04-13 13-39-51
Screenshot from 2020-04-13 13-40-11

Original on medium:
Screenshot from 2020-04-13 13-39-59
Screenshot from 2020-04-13 13-40-16

Yours on mobile:
Screenshot from 2020-04-13 13-40-16
Screenshot from 2020-04-13 13-41-06
Screenshot from 2020-04-13 13-41-13

Original on mobile:
Screenshot from 2020-04-13 13-41-20
Screenshot from 2020-04-13 13-41-26
Screenshot from 2020-04-13 13-41-31

Also, check the main three images, because when you change screen sizes, the images lose the right size.

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Use semantic HTML in the structure of your HTML file. You are doing great so far, but you could also include the `<article>` tag.

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version. You have some of them, but others are missing.



## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

File: index.html (hunk 1 above)
Comment (ishanchawla1) [reply index.html:32]: The Logo appears on the original Page on the Large Screen. It appears when you scroll down a few lines. I tried to find how to show it on scroll but couldn't find a way to do it without Javascript.

**Comment by uryelah** [comment]
Hello **Ishan Chawla**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**Good job with most of the previous fixes, the site is very responsive and good looking! 🌟**

There's just a small detail left from the last review. So read it carefully below and I'm sure it will be a piece of cake! 💪

---------     

## Required Changes 🏗 👷‍♀ 

- [ ] At the previous review my colleague meant that in medium screens the logo disappears from over the main article image. Compare below how yours still has the logo and gradient over it on medium-size screens and the original site does not:
Screenshot_2020-04-13 TNW
Screenshot_2020-04-13 TNW - International technology news, business and culture

It is disappearing on small screens but make sure the logo over the article disappears on medium screens too.

-----

- [ ] The mobile version of the site has a hamburger menu on the rightmost of the navbar. Check how it looks below:
Screenshot_2020-04-13 TNW - International technology news, business and culture(1)

Remember to include it on your clone too.

----

- [ ] Great job with the hover interactions on the `a` tags, just remember to also add a `href` value for them, it can be '#', so the cursor changes on hover too. (PS: don't forget to change the default link color after that).
------

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi @ishanchawla1, you have done a great job working on the changes pointed out by the previous TSE's 👍 Nice work. But before we proceed, let us address the comments below: These changes were highlighted by the previous TSE's
- ```Nav on mobile device```
```Currently```
mn1
```Original```
mn
- ```Footer on mobile device```, currently ours is a little packed. Let us compare ours with the original and make the changes
```Currently```
mf1
```Original```
mf
- ```Mobile screen``` Let us address the margin in the following. Kindly compare images
```Currently```
ml
```Original```
ml1
- ```Mobile screen``` Let us remove the vertical bar. Kindly compare images
```Currently```
mb
```Original```
mb1
- ```Tablet mode``` Let us fix the differences between the images below
```Currently```
mt
```Original```
mt1
```Currently```
mt3
```Original```
mt2


Kindly make these changes and submit another code review 😄 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Review by men32z — CHANGES_REQUESTED** [review]
Hey team!
## Good job on this project! 👍
#### and great job making the changes suggested by previous TSE's 
#### There are still some changes 👇 you need to implement to approve this request 🔍

Take a general look in all sizes and try to see how you can improve all the breakpoints and responsive content in general, paying attention to detail will improve your "pixel perfect" skills remember that this is a clone project and should be as closest as possible to the original.
kindly fix the requirements pointed by previous TSE's  👇

- [ ] as mentioned by previous TSE Mobile screen Let us address the margin in the following. Kindly compare images, also check that there are no tags in the images and they are smaller.
Currently

Original


- [ ] as mentioned by previous TSE Footer on mobile device, currently ours is a little packed. Let us compare ours with the original and make the changes
Currently

Original


- [ ] as mentioned by previous TSE The section is too wide compared to the original. That makes all the elements have larger dimensions. Please make the necessary adjustments.
( here the trick is to set a max-width property in the container div. so wider screens will have a limit.)
Currently 

Original



## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! :hammer:

--Luis Preza

**Comment by uryelah** [comment]
Hello **Ishan Chawla**, I'm Sarah, your reviewer for this PR 👩‍🚀  

**Good job with most of the fixes but it seems that we still need to implement some missing ones to meet the required responsiveness of this project 🌟**

---------     

## Required Changes 🏗 👷‍♀ 

[IMPORTANT] This was not fixed from the last reviews:
- [ ] At the previous review my colleague meant that in medium screens the logo disappears from over the main article image. Compare below how yours still has the logo and gradient over it on medium-size screens and the original site does not:
Screenshot_2020-04-13 TNW
Screenshot_2020-04-13 TNW - International technology news, business and culture

It is disappearing on small screens but make sure the logo over the article disappears on medium screens too.

-----

- [ ] The hamburger menu should only appear in the mobile version of the site. It is showing up in all screen sizes at the moment.

**Your navbar on wide screens:**
Screenshot_2020-04-15 TNW(2)

**How it should look instead:**
Screenshot_2020-04-15 TNW - International technology news, business and culture

----

- [ ]  The 'PopulateArticle' sections are breaking their image positioning on small devices:
Screenshot_2020-04-15 TNW

Make sure the top image does not repeat itself on small screens and the bottom one stays at the right of its container.

------

**Kindly fix the REQUIRED issues and send another code review request.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi @ishanchawla1, you have done a great job 👍 congratulations on fixing all but one of the changes requested by the previous reviewers 😄 We could even make our code even better. Kindly check comment below:

```In large screen device```, let us compare the following images and hide the necessary element
```Currently```
show2
```Original```
show3
```Currently```
show1
```Original```
show
Kindly show the hamburger icon on a small screen

Kindly make these changes and submit another code review 🙂 
_Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Pull request merged**